rust/windows-client/src-tauri/src/client/crash_handling.rs

```diff
@@ -80,8 +86,10 @@ fn start_server_and_connect() -> anyhow::Result<(minidumper::Client, std::proces
         }
 
         server = Some(
+            //
```

Comment:
Typo?

Reply:
Yeah. I was probably about to write `// TODO: pass the socket name to the server so we don't have to generate it in two places` and then just implemented that and left the slashes.